acnh.sql

```diff
@@ -5,6 +5,17 @@ CREATE TABLE IF NOT EXISTS user (
 	timezone TEXT
 );
 
+CREATE TABLE IF NOT EXISTS buy_price (
+	user_id TEXT,
+	week_local TEXT,
+	expiration TEXT,
```

Comment:
do we actually need expiration?

Reply:
I think it's worth keeping expiration since we're spread across a few different timezones for maximal bargain hunting

Reply:
but for any given `week_local`, you can translate it to UTC expiration without doing timezone math

Reply:
hmmm, but what if I forget to buy turnips and it's 1pm my time. Rushing in to !stalks buy and getting your and ari's listing would be more convenient than getting all of them.

Reply:
ohhhhhhh whoops, I've been thinking about this in terms of turnipprophet. forgot about that!